assets/js/feedback.js

```diff
@@ -2,12 +2,31 @@ const feedbackButtons = document.querySelectorAll('.feedback-button');
 
 feedbackButtons.forEach((button) => {
     button.addEventListener('click', function onClick(event) {
-        var url = window.location.href;
-        console.log(url);
-        console.log(button.value);
-        gtag('event', 'feedback', { 
-            'event_category': url, 
-            'event_action': button.value
+        gtag('event', 'feedback_click', { 
+            'helpful': button.value
         });
+        // find the hidden feedback text
+        sibling = button.nextElementSibling;
+
+        while (!(sibling.classList.contains('text-small') && sibling.classList.contains('hidden'))) {
+            sibling = sibling.nextElementSibling;
+        }
+        // show the hidden feedback text
+        if (sibling != null) {
+            sibling.classList.remove('hidden');
+        }
+
+        // disable the feedback buttons
+        button.disabled = true;
+        var buttonSibling;
+        if (button.id === 'yes') {
+            buttonSibling = button.nextElementSibling;
+        }
+        else {
+            buttonSibling = button.previousElementSibling;
+        }
+        if (buttonSibling != null) {
+            buttonSibling.disabled = true;
+        }
```

Comment:
Shouldn't this be `buttonSibling.disabled = false`? Also

```suggestion
        button[button.id === 'yes' ? 'nextElementSibling' : 'previousElementSibling']?.disabled = false;

Reply:
It should disable both buttons, so should be disabled=true